arch/riscv/core/isr.S

```diff
@@ -60,11 +60,19 @@
 	RV_E(	op ra, __z_arch_esf_t_ra_OFFSET(sp)	)
 
 #ifdef CONFIG_SMP
+#ifdef CONFIG_RV_BASE_CPU
 #define GET_CURRENT_CPU(dst, tmp) \
 	csrr tmp, mhartid				;\
+	addi tmp, tmp, -CONFIG_RV_BASE_CPU 	;\
```

Comment:
This begs that `CONFIG_RV_BASE_CPU` defaults to 0 here, since both declaration of the `GET_CURRENT_CPU` macro are basically identical.